build/azure-pipelines/win32/sql-product-build-win32.yml

```diff
@@ -28,6 +28,7 @@ steps:
       . build/azure-pipelines/win32/exec.ps1
       $ErrorActionPreference = "Stop"
       exec { tar -xf $(Pipeline.Workspace)/compilation.tar.gz }
+      exec { rm $(Pipeline.Workspace)/compilation.tar.gz }
```

Comment:
```suggestion
      # Delete compiled tarball now that we've extracted the files since it takes up a lot of space
      exec { rm $(Pipeline.Workspace)/compilation.tar.gz }
```